src/mol-canvas3d/canvas3d.ts

```diff
@@ -602,7 +602,8 @@ namespace Canvas3D {
                 const focus = camera.getFocus(center, radius);
                 const next = typeof nextCameraResetSnapshot === 'function' ? nextCameraResetSnapshot(scene, camera) : nextCameraResetSnapshot;
                 const snapshot = next ? { ...focus, ...next } : focus;
-                camera.setState({ ...snapshot, radiusMax: getSceneRadius() }, duration);
+                const offset = snapshot.target ? Vec3.distance(snapshot.target, scene.boundingSphere.center) : 0;
```

Comment:
Hmm, could you set `sceneRadiusFactor` instead? This definitely breaks things.

Reply:
Thanks, that makes more sense.

Reply:
Do you mean like this?: https://github.com/molstar/molstar/pull/1101/commits/00ad11dd6b170ad5815c12588dbd173aca47b01d

A disadvantage of tweaking `sceneRadiusFactor` is that it then stays changed forever (`radiusMax` in the original solution would go back to normal on the next camera reset).

Reply:
I suppose if you don't want the aggressive clipping, you want it forever?

Also note that you have access to the camera object and could change things there from your extension.

Reply:
The issue this is trying to solve it this:

![image](https://github.com/molstar/molstar/assets/21145776/ef0d6064-d701-46ab-81a5-42f81a7891ef)

Without either allowing bigger `radiusMax` or increasing the factor it seems quite hard to fix this, where the use case is loading a different state without moving the camera.

Any other ideas how to solve this @arose?

Reply:
> Any other ideas how to solve this @arose?

nothing at the moment